src/components/requests/features.info.vue

```diff
@@ -34,6 +51,30 @@ export default {
       let documentPathEnd = 'mapping/mapmodule/request/GetFeatureInfoRequest.md';
       e.target.href = this.$root.documentPathRequest + documentPathEnd;
     }
+  },
+  mounted () {
+    if (this.$root.channel && !layerVisibility) {
+      this.mapLayerVisibilityRequest(true);
+    }
+    listeners.push(EVENTBUS.on('channel.available', (event) => {
+      if (!layerVisibility) {
+        this.mapLayerVisibilityRequest(true);
+      }
+    }));
+  },
+  updated () {
+
+  },
+  beforeDestroy () {
+    this.mapLayerVisibilityRequest(false);
+    while (listeners.length) {
+      EVENTBUS.off('rpcAppDisplayMessage', listeners.pop());
+    }
   }
 }
+
+const toggleLayerVisibility = () => {
+  this.$root.channel.postRequest('MapModulePlugin.MapLayerVisibilityRequest', [492, true]);
+  this.$root.channel.log('MapModulePlugin.MapLayerVisibilityRequest posted with data', [492, true]);
+};
```

Comment:
This is not used at all? And probably won't work since this.$root doesn't work here?